src/autocomplete/EmojiProvider.tsx

```diff
@@ -69,15 +66,13 @@ export default class EmojiProvider extends AutocompleteProvider {
 
     constructor() {
         super(EMOJI_REGEX);
-        this.matcher = new QueryMatcher<IEmojiShort>(EMOJI_SHORTNAMES, {
-            keys: ['emoji.emoticon', 'shortname'],
-            funcs: [
-                (o) => o.emoji.shortcodes.length > 1 ? o.emoji.shortcodes.slice(1).map(s => `:${s}:`).join(" ") : "", // aliases
-            ],
+        this.matcher = new QueryMatcher<IEmojiShort>(EMOJI_SHORTCODES, {
+            keys: ['emoji.emoticon'],
+            funcs: [o => o.emoji.shortcodes.map(s => `:${s}:`).join(" ")],
             // For matching against ascii equivalents
             shouldMatchWordsOnly: false,
         });
-        this.nameMatcher = new QueryMatcher(EMOJI_SHORTNAMES, {
+        this.nameMatcher = new QueryMatcher(EMOJI_SHORTCODES, {
             keys: ['emoji.annotation'],
```

Comment:
so this matcher matches on `annotation` but EMOJI_SHORTCODES no longer has any results with 0 shortcodes